server/src/main/scala/controllers/application/AuthController.scala

```diff
@@ -51,7 +52,15 @@ abstract class AuthController(cc: ControllerComponents, ctrl: DrtSystemInterface
 
   def trackUser = Action.async { request =>
     val loggedInUser = ctrl.getLoggedInUser(config, request.headers, request.session)
-    ctrl.userService.insertOrUpdateUser(loggedInUser, None, None)
+    ctrl.userService.upsertUser(
+      UserRow(id = loggedInUser.id,
+        username = loggedInUser.userName,
+        email = loggedInUser.email,
+        latest_login = new java.sql.Timestamp(new java.util.Date().getTime),
```

Comment:
Could we use `ctrl.now().millisSinceEpoch` to get the milliseconds for consistency with the rest of the app?